README.md

```diff
@@ -933,16 +1003,176 @@ SUB COMMANDS:
+
+OPTIONS:
+    Row/field options
+        -c <copies>
+            Read multiple redundant values
+        -r, --raw
+            Get raw 24 bit values
+        -e, --ecc
+            Use error correction
+        -n, --no-descriptions
+            Don't show descriptions
+        -i <filename>
+            Include extra otp definition
+    Row/Field Selection
+        -z, --fuzzy
+            Allow fuzzy name searches in selector vs exact match
+        <selector>
+            The row/field selector, each of which can select a whole row:
+
+            ROW_NAME to select a whole row by name.
+            ROW_NUMBER to select a whole row by number.
+            PAGE:PAGE_ROW_NUMBER to select a whole row by page and number within page.
+
+            ... or can select a single field/subset of a row (where REG_SEL is one of the above row selectors):
+
+            REG_SEL.FIELD_NAME to select a field within a row by name.
+            REG_SEL.n-m to select a range of bits within a row.
+            REG_SEL.n to select a single bit within a row.
+            .FIELD_NAME to select any row's field by name.
+
+            .. or can selected multiple rows by using blank or '*' for PAGE or PAGE_ROW_NUMBER
+
+TARGET SELECTION:
+    Target device selection
+        --bus <bus>
+            Filter devices by USB bus number
+        --address <addr>
+            Filter devices by USB device address
+        --vid <vid>
+            Filter by vendor id
+        --pid <pid>
+            Filter by product id
+        --ser <ser>
+            Filter by serial number
+        -f, --force
+            Force a device not in BOOTSEL mode but running compatible code to reset so the command can be executed. After executing the
+            command (unless the command itself is a 'reboot') the device will be rebooted back to application mode
+        -F, --force-no-reboot
+            Force a device not in BOOTSEL mode but running compatible code to reset so the command can be executed. After executing the
+            command (unless the command itself is a 'reboot') the device will be left connected and accessible to picotool, but without the
+            RPI-RP2 drive mounted
+```
+
+```text
+$ picotool help otp set
+OTP SET:
+    Set the value of an OTP row/field
+
+SYNOPSIS:
+    picotool otp set [-c <copies>] [-r] [-e] [-s] [-i <filename>] [-z] <selector> <value> [device-selection]
+
+OPTIONS:
+    Redundancy/Error Correction Overrides
+        -c <copies>
+            Read multiple redundant values
+        -r, --raw
+            Set raw 24 bit values
+        -e, --ecc
+            Use error correction
+        -s, --set-bits
+            Set bits only
+        -i <filename>
+            Include extra otp definition
+        <value>
+            The value to set
+    Row/Field Selection
+        -z, --fuzzy
+            Allow fuzzy name searches in selector vs exact match
+        <selector>
```

Comment:
Is the `selector` functionality here the same as for `otp get`? In which case perhaps the corresponding help-text should be the same too? (I'll let you decide which of the two variants is better)